addon/emailTemplates.jsx

```diff
@@ -37,12 +37,24 @@ this.emailTemplates = (function () {
     return trunc;
   }
 
+  class Signature extends React.Component {
+    render() {
+      return (<div>
+              <br/>
+              ------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
I don't know ~~much~~ anything, about HTML-flavored emails, but is this possible to do with an `<hr/>` or CSS border?

Reply:
yeah hr would be much better

Reply:
For whatever reason Gmail seems to encourage a simple `--` as the signature separator. I'm not sure we actually need any separator? It might make things look weirder when people do have signatures.